kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/tasks/reports/KoverFormatCoverageTask.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2017-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.kover.gradle.plugin.tasks.reports
+
+import kotlinx.kover.gradle.plugin.dsl.AggregationType
+import kotlinx.kover.gradle.plugin.dsl.GroupingEntityType
+import kotlinx.kover.gradle.plugin.dsl.MetricType
+import kotlinx.kover.gradle.plugin.tools.CoverageRequest
+import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.*
+import javax.annotation.Nullable
+
+@CacheableTask
+internal abstract class KoverFormatCoverageTask : AbstractKoverReportTask() {
+    @get:Input
+    @get:Optional
+    @get:Nullable
+    abstract val header: Property<String>
+
+    @get:Input
+    abstract val lineFormat: Property<String>
+
+    @get:Input
+    abstract val groupBy: Property<GroupingEntityType>
+
+    @get:Input
+    abstract val coverageUnits: Property<MetricType>
+
+    @get:Input
+    abstract val aggregationForGroup: Property<AggregationType>
+
+    @get:OutputFile
+    abstract val outputFile: RegularFileProperty
```

Comment:
Should this be configurable via `KoverLogReportConfig` or is it some internal setting? Is this file supposed to be seen by user?

Reply:
These are implementation details, this task is intended only for printing to the log. With the appropriate request, we can add it later